apps/game_backend/lib/game_backend/matches.ex

```diff
@@ -114,6 +100,40 @@ defmodule GameBackend.Matches do
     end)
   end
 
+  defp complete_or_fail_bounties(multi, results) do
+    Enum.filter(results, fn result -> result["bounty_quest_id"] != nil end)
+    |> Enum.reduce(multi, fn result, multi ->
+      Multi.run(multi, {:complete_of_fail_bounty, result["user_id"]}, fn repo,
```

Comment:
This should be `complete_or_fail_bounty` ?

Reply:
yeah! missed typo